build-tests/arm/fedora/test-image-live/appliance.kiwi

```diff
@@ -17,7 +17,7 @@
         <rpm-check-signatures>false</rpm-check-signatures>
     </preferences>
     <preferences>
-        <type image="iso" flags="overlay" firmware="uefi" hybridpersistent_filesystem="ext4" hybridpersistent="true"/>
+        <type image="iso" flags="overlay" firmware="uefi" hybridpersistent_filesystem="ext4" hybridpersistent="true" kernelcmdline="console=ttyS0"/>
```

Comment:
Shouldn't it also include the `<bootloader name="grub2" console="serial"/>`? This is leading to a grub2 installation based on templates, as far as I remember the templates also support this feature.

Reply:
done